extensions/data-workspace/src/dataworkspace.d.ts

```diff
@@ -115,4 +127,21 @@ declare module 'dataworkspace' {
 		 */
 		element: any;
 	}
+
+	export interface IProjectAction {
+		/**
+		 * id of the project action
+		 */
+		readonly id: string;
```

Comment:
do we really need this?

Reply:
Can be skipped.. I will get rid of it.

Reply:
Removed displayName here and using id instead.